x-pack/plugin/ent-search/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/entsearch/260_rule_query_search.yml

```diff
@@ -239,7 +239,7 @@ setup:
 ---
 "Perform a rule query with an organic query that must be rewritten to another query type":
   - skip:
-      version: " - 8.13.99"
+      version: " - 8.12.2"
```

Comment:
We want this test to run on 8.12.2, correct? For my own learning, are these skip versions exclusive or inclusive? i.e. should this be `- 8.12.1` if we want the test to run on 8.12.2?

Reply:
Right, we want it to run on 8.12.2. Check this [doc](https://github.com/elastic/elasticsearch/blob/main/rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/README.asciidoc#skip-for-elasticsearch-versions) out for a good explanation.